src/app/components/recipes-and-filters/recipes-and-filters.component.ts

```diff
@@ -33,4 +41,33 @@ export class RecipesAndFiltersComponent {
     if (filters.limit) { r.limit = filters.limit };
     return r;
   }
+
+  /**
+   * Update the sidenav property for the current window.
+   */
+  updateSidenavForWindow() {
+    if (isPlatformBrowser) {
+      if (window.innerWidth < 768) {
+        this.navMode = 'over';
+        this.sidenav.close();
+      } else {
+        this.sidenav.open();
+      }
+    }
+  }
+
+  /**
+   * Listen to window resize.
+   */
+  @HostListener('window:resize', ['$event'])
```

Comment:
related https://github.com/angular/angular/issues/15000
although its scroll, if one is supported the other should be too.